core/web/resolver/mutation.go

```diff
@@ -592,3 +594,38 @@ func (r *Resolver) executeJobProposalAction(ctx context.Context, action jobPropo
 
 	return jp, nil
 }
+
+func (r *Resolver) UpdateUserPassword(ctx context.Context, args struct {
+	Input UpdatePasswordInput
+}) (*UpdatePasswordPayloadResolver, error) {
+	if err := authenticateUser(ctx); err != nil {
+		return nil, err
+	}
+
+	session, ok := auth.GetGQLAuthenticatedSession(ctx)
+	if !ok {
+		return nil, errors.New("couldn't retrieve user session")
```

Comment:
This error path needs a test

Reply:
This one will never get reached, since the first authentication check will prevent the code from even arriving here...

maybe we just ignore the `ok` flag in this case?

Reply:
Fair enough. This should be fine